documents/LORE.md

```diff
@@ -0,0 +1,25 @@
+# The Stickian Empire
+
+## Topics
```

Comment:
Should not be topics, this is a story, at most it's a summary. So drop down the numbers (for the official md you are going to merge here) and keep a nice summary of our story here.

Reply:
If you read the initial comment you'd understand that before making the first draft it is important to define the story key points, and one way of doing that it's with topics. Moreover, as also mentioned, this is a WIP for the draft, so this kind of comments are really unnecessary...

Reply:
@luisferreira32 if you were paying attention to the meetings you'd know.